SYCL/DeviceCodeSplit/double-grf.cpp

```diff
@@ -0,0 +1,139 @@
+//==----------- double_grf.cpp  - DPC++ SYCL on-device test ---------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// This test verifies effect of
+//   set_kernel_properties(kernel_properties::use_double_grf);
+// API call in device code:
+// - ESIMD/SYCL splitting happens as usual
+// - SYCL module is further split into callgraphs for entry points requesting
+//   "double GRF" and callgraphs for entry points which are not
+// - SYCL device binary images requesting "double GRF" must be compiled with
+//   -doubleGRF option
```

Comment:
Is this the correct option?

Reply:
good eye, it's technically right but it's not what we use anymore. fixed in latest commit